### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by nichtsfrei**
- **Fix: add redis to the storage options for clap**
- **Add: env REDIS_URL to set redis-url**
- Fix: configure different redis url within openvas and openvasd 
- Fix: not rescheduling on openvas panic

SC-1048

### CODE REVIEW

**Review by jjnicola — CHANGES_REQUESTED** [review]
rust/openvasd/README.md needs to be updated with the new storage type and the change from `wrapper-` to `scanner-type`.

**Comment by nichtsfrei** [comment]
> rust/openvasd/README.md needs to be updated with the new storage type and the change from `wrapper-` to `scanner-type`.

fixed

File: rust/openvasd/README.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ If the signature check is enabled, it is also required to set the the `GNUPGHOME
...
 | API key                  | --api-key               |               | endpoints                          | key               | API_KEY                  | API key that must be set as X-API-KEY header to gain access. If none is given, api-key authorization is disabled                                                          |                               |
-| Wrapper Type             | --wrapper-type          |               | scanner                            | type              | WRAPPER_TYPE             | Type of wrapper used to manage scans, currently only `OSPD` is available                                                                                                  | OSPD                          |
+| Wrapper Type             | --scanner-type          |               | scanner                            | type              | WRAPPER_TYPE             | Type of wrapper used to manage scans, currently only `OSPD` is available                                                                                                  | OSPD                          |
```
Comment (jjnicola) [inline rust/openvasd/README.md:204]:
```suggestion
| Scanner Type             | --scanner-type          |               | scanner                            | type              | SCANNER_TYPE             | Type of wrapper used to manage scans, currently `openvas` and `ospd` are available                                                                                                  | ospd                          |
```

Comment (jjnicola) [reply rust/openvasd/README.md:204]: missing option and fix EnvVar

Comment (jjnicola) [reply rust/openvasd/README.md:204]: Also missing  the new `REDIS_URL` environment variable in the options table.

**Review by jjnicola — CHANGES_REQUESTED** [review]

**Review by jjnicola — APPROVED** [review]

**Pull request merged**